angularjs-portal-home/src/main/webapp/my-app/layout/partials/home-toggle.html

```diff
@@ -5,6 +5,7 @@
       tooltip-trigger="mouseenter"
       tooltip-placement="top"
       tooltip-popup-delay="500"
+      onClick="_gaq.push(['_trackEvent', 'Widgets', 'List Mode', 'List Mode']);"
```

Comment:
Oh interesting, that service is pretty easy. I'll use that. I was going off of the Beta event in beta-header.html:
`<a onClick="_gaq.push(['_trackEvent', 'Beta', 'Read About Change', 'Read About Change']);" 
	          	   href="http://redesign.my.wisc.edu/" 
	          	   target="_blank">
	          		Read about what's changing
	          	</a>.`
Should I refactor this as well?